client/state/happychat/selectors.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import map from 'lodash/map';
+import { map, findLast } from 'lodash';
```

Comment:
not in Calypso. we have a Babel transform which separates out lodash so we actually prefer using this named import syntax with multiple functions in one line